paydb_endpoint.py

```diff
@@ -1,4 +1,5 @@
 # pylint: disable=unbalanced-tuple-unpacking
+# pylint: disable-msg=too-many-function-args
```

Comment:
i get the feeling this line is not needed

Reply:
deleted